docs/api.md

```diff
@@ -49,6 +49,13 @@ A `World` class is the class with all the specifics of a certain game to be
 included. It will be instantiated for each player that rolls a seed for that
 game.
 
+### WebWorld Class
+
+A `WebWorld` class contains specific attributes and methods that can be modified
+for your world specifically on the webhost. At the moment this comprises of `settings_page`
```

Comment:
List of attributes somewhere down in the implementation would be good, but i think the really important thing - the tracker - is not in WebWorld yet, right?

Reply:
Tracker is not part of webworld yet, correct. Do we want a section within the api.md for each class to detail its individual attributes and methods since I noticed nearly everything from Baseclasses.py is currently missing in this document and wasn't sure if it should be left alone and just pointed to or put in a separate doc.

Reply:
i wrote that somewhat in the style of a step-by-step with explanations in between. don't think it needs to be 100% complete, but if you come up with a potential scenario for something, feel free to add that where it's appropriate